src/LiquidationPair.sol

```diff
@@ -100,6 +103,14 @@ contract LiquidationPair {
       uint256(_virtualReserveIn) * _virtualReserveOut >= _minK,
       "LiquidationPair/virtual-reserves-greater-than-min-k"
     );
+    require(
+      UFixed32x4.unwrap(_maxPriceImpactOut) < 1e4,
+      "LiquidationPair/max-price-impact-greater-than-100"
+    );
+    require(
+      UFixed32x4.unwrap(_maxPriceImpactOut) >= 10,
+      "LiquidationPair/max-price-impact-less-than-1"
```

Comment:
Same idea here, the `1` in the error is referring to "1%" rather than the unformatted number. Maybe `LiquidationPair/max-price-impact-less-than-1-percent` makes the most sense.